src/renderer/services/neo.js

```diff
@@ -509,7 +509,14 @@ export default {
                         const existingHolding = this.getHolding(holding.assetId);
                         if (existingHolding) {
                           // TODO: May need some way in the UI to show that the balance of this asset may be out of date
-                          holdings.push(existingHolding);
+                          holding.balance = existingHolding.balance;
+                          holding.symbol = existingHolding.symbol;
+                          holding.name = existingHolding.name;
+                          holding.availableBalance = calculateHoldingAvailableBalance(holding);
+                          holding.totalBalance = calculateHoldingTotalBalance(holding);
+                          holding.decimals = val.decimals;
```

Comment:
shouldn't this be existingHolding.decimals

Reply:
Fixed